### ISSUE TITLE
Provide alternate image registry to Docker Hub

### ISSUE THREAD

**Issue by bjw-s** [tomsquest/docker-radicale#129]
Since Docker Hub has been applying rate limits on image pulls for a while now, and their more recent move to scrap free teams I feel that Docker Hub is becoming more and more hostile towards the OSS/hobbyist community.

I would like to propose providing GitHub Container Registry as an alternative (or completely switching over to it) for the project.

Would you be open to this? If so, I'd be willing to write up a PR to implement.

**Comment by tomsquest** [comment]
Hi @bjw-s ,

I read about that yesterday.

Somehow, the images we published are not concerned from what I know. Can you confirm?

And I would avoid pushing twice (dockerhub, gcr), and I don't know if people want to switch.

So I would say: as we don't have a problem, now, I don't think I want to _fix_ it. If the situation changes, yes definitely we would switch.

**Comment by clo-dev** [comment]
Hi @tomsquest,

Here's an official statement from Docker explaining the changes (Their first attempt was in fact, terrible):
https://www.docker.com/blog/we-apologize-we-did-a-terrible-job-announcing-the-end-of-docker-free-teams/

TLDR
Options:
- If you're using the **Free Team Org** plan then you'll need to upgrade to the pay plan.
- Apply for the sponsorship https://www.docker.com/community/open-source/application/
- Move to https://ghcr.io

Hope this helps you and others in making future decisions on this matter.

**Comment by tomsquest** [comment]
Hi @clo-dev ,

I am using a Personal plan, so "everything" is fine for now (not yet a problem).
If DockerHub becomes the new TravicCI, definitely I will switch (or github/microsoft will do first :/ ).
